docs/source/user_guide/user_guide/scalar_tensors.md

```diff
@@ -0,0 +1,11 @@
+# Scalar tensors
+
+To create a scalar tensor, use a shape of `()`. Note that this only works for fields, not ndarrays. (An NDArray needs to have at least one dimension).
+
+You can access the value in a scalar tensor by indexing with `[None]`:
+
+```
+@ti.kernel
+def f1(a: ti.Template) -> None:
+    a[None] += 1
```

Comment:
As a side note, it would be nice to support the same syntax as numpy or torch, i.e. `[()]`.